chart/templates/_commonChecks.tpl

```diff
@@ -218,6 +218,11 @@ Return common collectors for preflights and support-bundle
             emptyDir:
               sizeLimit: 8Mi
           {{- end }}
```

Comment:
I suppose that here you should add a check to avoid define the secret as value and as configmap as the same time, right?

Reply:
It's already checked :) If we define `.Values.externalProxy.sslCA` we define it as value, but if `.Values.externalProxy. sslCAConfigmapName` is defined then we define it as configmap

Reply:
Just added it, nice cath :)